index.js

```diff
@@ -84,10 +84,33 @@ server.post('/api/register/', (req, res) => {
     })
     .catch(err => res.status(500).send(err))
   })
-  .catch(err => {res.status(500).send(err);
-  })
+  .catch(err => res.status(500).send(err))
 })
 
+
+
+// server.post('/api/register/', (req, res) => {
+//   const user = req.body;
+//   console.log(`Tommy's session:`, req.session)
+
+//   user.password = bcrypt.hashSync(user.password, 14)
+//   db('users').insert(user)
+//   .then(ids => {
+//     const id = ids[0];
+//     // find the user using the ID
+//     db('users')
+//     .where({ id })
+//     .first()
+//     .then(user => {
+//       const token = generateToken(user);
+//       res.status(201).json({id: user.id, token})
+//     })
+//     .catch(err => res.status(500).send(err))
+//   })
+//   .catch(err => {res.status(500).send(err);
```

Comment:
Remove old code when you can (unless you're planning on using it later; then remove it once you're done with it if it ends up not being used)!